.travis.yml

```diff
@@ -1,14 +1,22 @@
+---
 language: python
 
 python:
-    - "2.7"
+  - 2.7
 
-install: "make"
+install:
+  - pip install -r requirements.txt
+  - pip install yamllint pep8
```

Comment:
Is this the way we should split build/run requirements from test/authortest requirements?